### PR TITLE
Docs monitoring update

### PR DESCRIPTION
**Pull request by gzhao408**
Addresses issue #43. Updates docs to reflect changes in GCP UI now that Stackdriver is now Operations, and all UI is embedded in GCP console.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -291,19 +291,19 @@ Stackdriver Monitoring ([documentation](https://cloud.google.com/monitoring/docs
 
 #### Using Monitoring
 
-To get to Stackdriver Monitoring from the GCP console, select **Monitoring** on the navigation panel. This brings up the Stackdriver Monitoring console, a separate UI from the consoles for other GCP and Stackdriver products.  By default, you reach an overview page:
+To get to Stackdriver Monitoring from the GCP console, select **Monitoring** on the navigation panel. By default, you reach an overview page:
```
Comment (simonz130) [inline docs/README.md:294]: It's not "Stackdriver Monitoring", it's "Cloud Monitoring". Please change throughout.

Comment (gzhao408) [reply docs/README.md:294]: Got it

**Comment by gzhao408** [comment]
Also just noticed the install script links to "Stackdriver Dashboard" (app.google.stackdriver.com/accounts/create) at the end, but this page no longer contains any information. I will replace with the GCP > Monitoring > Overview page.

**Comment by gzhao408** [comment]
Latest commit removes stackdriver dashboard from the install script display message and changes pretty much every instance of "Stackdriver" to "Operations," happy to revert if it's too much.

File: docs/README.md (hunk 1 above)
Comment (gzhao408) [reply docs/README.md:294]: I put this change in #57 since we discussed doing a full replacement as a separate task from this.

**Pull request merged**